packages/core/test/helpers/carv2-index.js

```diff
@@ -22,7 +22,9 @@ export const encode = async items => {
   return { cid: Link.create(MultihashIndexSortedWriter.codec, digest), bytes }
 }
 
-/** @param {Uint8Array} bytes */
+/**
```

Comment:
could we do this in `src`? not sure this will trigger release

Reply:
oops well spotted 👍